kbase-extension/static/kbase/js/widgets/appWidgets2/input/fileInput.js

```diff
@@ -10,41 +10,49 @@ define([
     'kb_service/client/userAndJobState',
     'kb_service/client/shock',
     '../validators/text',
+    '../validators/constants',
```

Comment:
This file has most of the sweeping changes, all of which were to appease ESLint.